packages/mattermost-redux/src/selectors/entities/users.ts

```diff
@@ -692,25 +692,27 @@ export function searchProfilesInGroup(state: GlobalState, groupId: Group['id'],
     return profiles;
 }
 
+export function checkIsFirstAdmin(currentUser: UserProfile, users: IDMappedObjects<UserProfile>): boolean {
+    if (!currentUser) {
+        return false;
+    }
+    if (!currentUser.roles.includes('system_admin')) {
+        return false;
+    }
+    const userIds = Object.keys(users);
+    for (const userId of userIds) {
+        const user = users[userId];
```

Comment:
Looks like this could be replaced by a `for (const user of users)` - do I miss something?

Reply:
We can almost do that. JS doesn't allow for...of object iteration , but we can iterate on the `Object.values` which I think is cleaner code and has the perks of being faster: https://jsbench.me/6il02vjfuw/1

Reply:
![image](https://user-images.githubusercontent.com/785518/155802350-9166e2c1-1847-4ae7-8607-4adc8b5f7a43.png)
Slight improvement on my machine, only +~525op/s haha